test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxserviceTests.cs

```diff
@@ -159,6 +160,40 @@ public async void ExtractDacpac()
             }
         }
 
+        /// <summary>
+        /// Verify the extract request to create Sql file
+        /// </summary>
+        [Fact]
+        public async void ExtractDBtoSqlFile()
+        {
+            var result = GetLiveAutoCompleteTestObjects();
+            SqlTestDb testdb = await SqlTestDb.CreateNewAsync(TestServerType.OnPrem, false, null, SourceScript, "DacFxExtracttoSqlTest");
+            string folderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "DacFxTest");
+            Directory.CreateDirectory(folderPath);
+
+            try
+            {
+                var extractParams = new ExtractParams
+                {
+                    DatabaseName = testdb.DatabaseName,
+                    PackageFilePath = Path.Combine(folderPath, string.Format("{0}.sql", testdb.DatabaseName)),
```

Comment:
Does this codebase support string interpolation yet?  Instead of `string.Format("{0}.sql", testdb.DatabaseName)`, we can do `$"{testdb.DatabaseName}.sql"`, which is cleaner and less prone to mistakes in future changes.

Reply:
Tried this, but didn't work. Probably some version needs to be updated.